test/handlers/preformat/shift-enter-last-line.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx h */
+
+export default input => input.break({ shift: true })
```

Comment:
When editing multiline blocks this would require you to press shift+enter multiple times when editing, but enter just once, and I think the more common action should also be the easier to invoke. 

And while I agree that this makes editing single line block slightly more difficult, I don't think saving one keystroke when editing single line blocks justifies making editing multiline block that inconvenient.

Reply:
We can discuss it as a part of: https://github.com/openstax-poland/cnx-designer/issues/91